syft/serde.py

```diff
@@ -1304,6 +1359,8 @@ def _simplify(obj: object) -> object:
         # for this type. If there is, run return
         # the simplified object
         current_type = type(obj)
+        if isinstance(obj, torch.jit.ScriptModule):
+            current_type = torch.jit.ScriptModule
 
```

Comment:
We should avoid avoid if/else condition in the `_simplify` which is really used a lot. I would suggest keep the type as an entry of the dict: `torch.jit.TopLevelTracedModule` or other: it is not a pb to have several keys with the same value in the simplify dict, but adding a if statement slower all the ser/deser process

Reply:
That is weird, this code was already removed. I will take it out again.

Reply:
I've just pulled the most recent version of this branch and is still here. I'll try to remove it now.

Reply:
done